assets/prototype/domain/eventEditor/containers/mutations/useDeleteTicketMutation.js

```diff
@@ -1,9 +1,45 @@
+import get from 'lodash/get';
 import { useMutation } from '@apollo/react-hooks';
 import { DELETE_TICKET } from './tickets';
+import { GET_TICKETS } from '../queries/tickets';
 
-const useDeleteTicketMutation = () => {
+const useDeleteTicketMutation = ({ datetimeIn, id }) => {
```

Comment:
It's better to generate `datetimeIn` right there in the hook instead of being passed down to it all the way from `TicketList`. It should only get the `id` and do the rest itself. What we could do later is to create another hook like `useDatetimeIds` to fetch the value to be used for `datetimeIn`.